dragonite/config.toml.example

```diff
@@ -36,14 +38,13 @@ bearer_token = "KOJI_SECRET"
 #enabled = true
 
 [tuning]
-#recycle_gmo_limit = 4900
-#recycle_encounter_limit = 9900
-#recycle_invasion_limit = 940
-#recycle_on_jail=false
-#minimum_account_reuse_hours = 169 # default 0
-#location_delay = 0
-#fort_location_delay = 0
-#scout_age_limit = 30
+#recycle_gmo_limit = 4990
+#recycle_encounter_limit = 9975
+#recycle_invasion_limit = 900  # Only works in testing builds at the moment.
```

Comment:
can you pls remove the comment? I will merge this as soon as we release production :)